src/app/organizations/manage/policies.component.html

```diff
@@ -1,3 +1,6 @@
+<button class="p-0 deprecationWarning" (click)="goToEnterprisePortal()">
+    <app-callout [type]="'warning'">{{'webPoliciesDeprecationWarning' | i18n}}</app-callout>
+</button>
```

Comment:
Can we follow this style for callout action buttons?

![image](https://user-images.githubusercontent.com/1190944/95578544-0977f180-0a02-11eb-90f3-f570c575a6a7.png)

Reply:
done